wgsl/index.bs

```diff
@@ -897,22 +897,65 @@ A <dfn noexport>type rule</dfn> has two parts:
     * How the expression is used in a statement.
     * Optionally, other static context.
 
-Each distinct type parameterization for a type rule is called an <dfn noexport>overload</dfn>.
-For example, [[#arithmetic-expr|unary negation]] (an expression of the form `-`|e|)
-has eight overloads, because its type rules are parameterized by a type |T| that can be any of:
-* [=i32=]
-* vec2&lt;i32&gt;
-* vec3&lt;i32&gt;
-* vec4&lt;i32&gt;
-* [=f32=]
-* vec2&lt;f32&gt;
-* vec3&lt;f32&gt;
-* vec4&lt;f32&gt;
+Type rules may have <dfn noexport>type parameters</dfn> in their preconditions and conclusions.
+When a type rule's conclusion or preconditions contain type parameters,
+we say it is <dfn noexport>parameterized</dfn>.
+When they do not, we say the rule is <dfn noexport>concrete</dfn>.
```

Comment:
Potential collision with another PR.

The PR for creation-time expressions defines "concrete" in a different way: a type is concrete if it is not an abstract numeric type and does not contain an abstract numeric type. [see here](https://github.com/gpuweb/gpuweb/pull/2668/files#diff-709b5c343d59ae10cbabe69f9df1eee9e2dbb6e9d848632fa94357083d6e0a34R1171)

That definition gets used in several other sections of the spec.

I suggest we change this term to "fully elaborated":

      When they do not, we say the rule is <dfn noexport>fully elaborated</dfn>.